<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-S8E8KO3" name="org.example.TestRunnerProject.report" tests="19" failures="2" timestamp="2022-10-11T21:18:03 EET" time="331.906" errors="1">
  <testcase name="runScenario" time="30.757" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="12.608" classname="org.example.TestRunnerProject.report">
    <failure type="java.lang.AssertionError" message="expected [https://demo.nopcommerce.com/iphone-6] but found [https://demo.nopcommerce.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://demo.nopcommerce.com/iphone-6] but found [https://demo.nopcommerce.com/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at org.example.StepDefinitions.D06_SlidersWrapperStepD.displayedFirstSlider(D06_SlidersWrapperStepD.java:30)
at ✽.relative product for first slider is displayed(file:///C:/Users/MakkaH-73/IdeaProjects/DemoNopCommerceProject/src/main/resources/ProjectFeatures/F06_SlidersWrapper.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="23.966" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="13.843" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="14.789" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="13.938" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="15.998" classname="org.example.TestRunnerProject.report">
    <failure type="java.lang.AssertionError" message="expected [https://demo.nopcommerce.com/nokia-lumia-1020] but found [https://demo.nopcommerce.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://demo.nopcommerce.com/nokia-lumia-1020] but found [https://demo.nopcommerce.com/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at org.example.StepDefinitions.D06_SlidersWrapperStepD.SecondSlider(D06_SlidersWrapperStepD.java:43)
at ✽.relative product for second slider is displayed(file:///C:/Users/MakkaH-73/IdeaProjects/DemoNopCommerceProject/src/main/resources/ProjectFeatures/F06_SlidersWrapper.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="13.873" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="17.768" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="18.901" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="13.741" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="14.568" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="16.123" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="26.530" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="11.227" classname="org.example.TestRunnerProject.report">
    <error type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
at java.util.ArrayList.rangeCheck(ArrayList.java:659)
at java.util.ArrayList.get(ArrayList.java:435)
at org.example.StepDefinitions.D07_FollowUsStepD.newTabOpened(D07_FollowUsStepD.java:46)
at ✽."https://demo.nopcommerce.com/new-online-store-is-open" The link is opened in new tab(file:///C:/Users/MakkaH-73/IdeaProjects/DemoNopCommerceProject/src/main/resources/ProjectFeatures/F07_FollowUs.feature:14)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="32.018" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="17.884" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="11.185" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
  <testcase name="runScenario" time="12.189" classname="org.example.TestRunnerProject.report"/>
  <system-out/>
</testsuite> <!-- org.example.TestRunnerProject.report -->
